src/main/scala/com/gu/salesforce/messageHandler/MessageHandlerLambda.scala

```diff
@@ -93,11 +93,11 @@ trait MessageHandler extends Logging {
       if (parsedMessage.getOrganizationId.startsWith(Config.salesforceOrganizationId)) {
         processNotifications(parsedMessage.getNotification.toList, outputStream)
       } else {
-        logger.info("Unexpected salesforce organization id in xml message")
+        logger.info(s"Unexpected salesforce organization id ${parsedMessage.getOrganizationId} in xml message. Expected: ${Config.salesforceOrganizationId}.")
```

Comment:
is it ok to log the organization id or would it be like logging a secret token?

Reply:
Ah yes, I don't need a pull request to deploy a build! I'll close the PR.